tests/e2e/html.test.ts

```diff
@@ -59,4 +59,21 @@ describe('HTML E2E', () => {
 
         expect(extractor.getPotString()).toBe(fs.readFileSync(__dirname + '/fixtures/html/embeddedJs.expected.pot').toString());
     });
+
+    test('line number start 11', () => {
+        const extractor = new GettextExtractor();
+        const jsParser = extractor.createJsParser([
+            JsExtractors.callExpression('__', { arguments: { text: 0, } }),
+            JsExtractors.callExpression('_n', { arguments: { text: 0, textPlural: 1 } }),
+            JsExtractors.callExpression('_xn', { arguments: { context: 0, text: 1, textPlural: 2 } }),
+        ]);
+        const htmlParser = extractor.createHtmlParser([
+            HtmlExtractors.embeddedAttrJs('[*=*]', jsParser),
+            HtmlExtractors.embeddedJs('*', jsParser),
```

Comment:
These unspecific selectors seem quite unrealistic to me. In real applications those would probably be narrowed down to certain elements / attributes, so I'd do the same here.